internal/util/update/resource-merge.go

```diff
@@ -99,3 +107,35 @@ func (u ResourceMergeUpdater) updatedKptfile(updatedPath string, options UpdateO
 	err = kf.MergeOpenAPI(options.KptFile)
 	return kf, err
 }
+
+// setAllSetterDefinitions reads all the Setter Definitions from Kptfile in local
+// package path and sets all setter values in original and updated folders for
+// resource merge to be accurate
+func setAllSetterDefinitions(packagePath, originalPath, updatedPath string) error {
```

Comment:
Follow up:

- Move this into the setters2 package in kustomize
- Make the paths a var args
-

Reply:
Done